src/test/java/com/madgag/gif/fmsware/TestAnimatedGifEncoder.java

```diff
@@ -83,6 +114,22 @@ private void encodeSampleSonicFrames() {
         encoder.finish();
     }
 
+    private void encodeSampleExactFrames() {
+        encoder.setRepeat(0);
+        encoder.setDelay(1000);
+        encoder.addFrame(agif);
+        encoder.addFrame(bgif);
+        encoder.finish();
+    }
+
+    private void encodeSampleExactFramesTransparent() {
```

Comment:
Well, you're right. This is because I created a different test but at the end of the day I realized it was not necessary, so I simplified the test case but I forgot to remove this auxiliary method. Now it has been removed.